### PR TITLE
Update GcSqlInstance

### PR DESCRIPTION
**Pull request by marceloyuela**
The Update GcSqlInstance cmdlet.

### CODE REVIEW

**Comment by marceloyuela** [comment]
I work at google

File: Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs [hunk 1]
```diff
@@ -987,4 +988,363 @@ protected override void ProcessRecord()
...
+                WaitForSqlOperation(result);
+            }
+            else
+            {
+                newSettings = PopulateSetting(body.Settings);
+                body.Settings = newSettings;
+                InstancesResource.PatchRequest request = Service.Instances.Patch(body, project, instance);
+                Operation result = request.Execute();
+                WaitForSqlOperation(result);
+            }
```
Comment (marceloyuela) [inline Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs]: Do you think users would like the updated Instance to be returned?

Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs]: That is a good idea.

**Comment by marceloyuela** [comment]
Returns the instance now

File: Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs [hunk 2]
```diff
@@ -987,4 +988,387 @@ protected override void ProcessRecord()
...
+        /// <summary>
+        /// <para type="description">
+        /// The tier of service for this instance, for example "db-n1-standard-1".
+        /// Pricing information is available at https://cloud.google.com/sql/pricing.
+        /// Get-GcSqlTiers will also tell you what tiers are available for your project.
+        /// </para>
+        /// </summary>
+        [Parameter(Position = 1, Mandatory = true, ParameterSetName = ParameterSetNames.ByName)]
+        [Parameter(Position = 0, Mandatory = true, ParameterSetName = ParameterSetNames.ByInstance)]
+        public string Tier { get; set; }
```
Comment (ILMTitan) [inline Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs]: Why is this mandatory?

Comment (marceloyuela) [inline Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs]: If it's not specified the operation fails, but I suppose I could pull it from the instance.

**Comment by marceloyuela** [comment]
Changed the above issue and removed the database cmdlet- as discussed in person.

**Comment by ILMTitan** [comment]
LGTM

**Pull request merged**